tests/network/vmi_multus.go

```diff
@@ -258,6 +271,30 @@ var _ = Describe(SIG("Multus", Serial, decorators.Multus, func() {
 				By("checking virtual machine instance can ping using ptp cni plugin")
 				Expect(libnet.PingFromVMConsole(detachedVMI, multiIfaceNetGatewayIP2)).To(Succeed())
 			})
+
+			It("VirtualMachineInstance with cni ptp plugin interface with custom MAC address", func() {
+				By("Verifying the desired custom MAC is the one that was actually configured on the interface.")
+				ipLinkShow := fmt.Sprintf("ip link show eth2 | grep -i \"%s\" | wc -l\n", customMacAddress)
+				err = console.SafeExpectBatch(detachedVMI, []expect.Batcher{
+					&expect.BSnd{S: ipLinkShow},
+					&expect.BExp{R: "1"},
+				}, 15)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Verifying the desired custom MAC is not configured inside the pod namespace.")
+				vmiPod, err := libpod.GetPodByVirtualMachineInstance(detachedVMI, detachedVMI.Namespace)
+				Expect(err).NotTo(HaveOccurred())
+
+				//echo -n "ptp3" | sha256sum | cut -c1-11
```

Comment:
Instead comment, maybe lets implement it please with go ?
This way it will be self heal in case of changes, like this PR introduced,
and also self explanatory

(please use ptp3 as const, worth to make all ethN / ptpN consts if possible)
```
	hash := sha256.Sum256([]byte("ptp3"))
	hexStr := hex.EncodeToString(hash[:])
	fmt.Println(hexStr[:11])
```

Reply:
not that the test will print the hash, but that it will calculate it, this is just ref code
```
	hash := sha256.Sum256([]byte("ptp3"))
	hexStr := hex.EncodeToString(hash[:])
	myHash := hexStr[:11]
```
and then use `myHash`

This will reduce the need to have comment and also self heal in case someone changes the network name, same as you had to do